src/platform/silabs/wifi/rs911x/WifiInterfaceImpl.h

```diff
@@ -28,6 +28,7 @@ namespace Silabs {
 class WifiInterfaceImpl final : public WiseconnectWifiInterface
 {
 public:
+    static constexpr uint32_t kDhcpPollIntervalMs = 250;
```

Comment:
This doesn't need to be in the class, it can go in the anounymous namespace in the .cpp

Reply:
Moved to the namespace